cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +111,60 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	cmds.SetHelpCommand(newKubeadmHelpCmd(cmds.Name()))
+	cmds.SetUsageTemplate(cmds.UsageTemplate() + kubeadmUsageTemplateRequiredImages)
 	return cmds
 }
+
+// newKubeadmHelpCmd returns a help command for kubeadm root command.
+// see cobra.Command#initHelpCmd() (command.go)
+func newKubeadmHelpCmd(cmdName string) *cobra.Command {
+	return &cobra.Command{
+		Use:   "help [command]",
+		Short: "Help about any command",
+		Long: `Help provides help for any command in the application.
```

Comment:
except it doesn't. update the help help text to be accurate about what subcommands it currently supports.

why is `kubeadm help cmd` different to `kubeadm cmd --help`